public_html/css/style.css

```
@@ -887,3 +887,13 @@ body {
     opacity: 0.5;
     pointer-events: none;
 }
+
+/* StyleCodes (MARKUP: StyleCodes 1.0) */
+.sc-body {
+    white-space: pre-wrap;
+    word-wrap: break-word;
```

Stylelint (17.3.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)